src/SlackFunction.ts

```diff
@@ -202,9 +205,61 @@ export class SlackFunction {
   }
 
   /**
-   * Attach a block_suggestion interactivity handler to your SlackFunction
+   * Attach a block_actions interactivity handler to your SlackFunction.
+   * This function is added as an alias for the action() function to create
+   * consistency with having a blockSuggestion() function.
+   *
+   * NOTE: This function is a direct copy of the action() function and
```

Comment:
Is there any way to use the .action() function within blockAction instead of copying the code? If we can do this, it would probably reduce the likelihood of accidental drift between the two interfaces, since we expect them to be the same.

Reply:
absolutely! I'll try what you suggested above and have this updated by later today! 🙌